helm/hpcc/values.yaml

```diff
@@ -38,6 +58,65 @@ global:
     moneyLocale: "en_US.UTF-8"
     # perCpu: 0.126
 
+##Local instance of Elastic stack will be deployed alongside HPCC components
+##Elastic stack is comprised of several components which follow below and can be customized
+##by overwriting/setting helm chart values here
+
+##The elasticsearch component can be customized by modifying helm chart values here.
+elasticsearch:
+  description: "HPCC Managed Elasticsearch"
+  ##See https://github.com/elastic/helm-charts/blob/master/elasticsearch/values.yaml for all available options
+  #antiAffinity: "soft"
+  #replicas: 2
+  labels: {"managed" : "HPCC"}
+  #volumeClaimTemplate:
```

Comment:
@ghalliday this is where we can customize the elasticsearch helm values, the commented out values illustrate the default values. Does it make sense to somehow tie this into the storage planes?

Reply:
The storage planes are primarily for defining places where data is written directly by the platform.  I don't think this needs to be tied in to that.